niceml/experiments/expfilenames.py

```diff
@@ -28,6 +28,7 @@ class ExperimentFilenames:  # pylint: disable=too-few-public-methods
     NET_DATA_FOLDER: str = "net_data"
     EXP_FILES_FILE: str = "exp_files.parq"
     EXP_FILES_COL: str = "exp_files"
+    SUBSET_NAME: str = "{set_name}.parq"
```

Comment:
is the 'set_name' the same as the 'dataset_name', which is referred to in Line 22? if so, please use consistent naming.

Reply:
It means the same, but at this point it must be named the same as the "set_name" attribute of the DfDataset class.

Reply:
should we rename the attribute in the DfDataset then? Otherwise we are using different terms for the same thing in our project.

Reply:
I would recommend renaming both of them into subset_name. I think this fits best